radeon-profile/gpu.cpp

```diff
@@ -131,7 +131,7 @@ void gpu::defineAvailableDataContainer() {
 
     GPUPwmStruct tmpPWm = driverHandler->getPwmSpeed();
 
-    if (tmpPWm.pwmSpeed != -1)
+    if (tmpPWm.pwmSpeed != -1 && gpuParams.pwmMaxSpeed != -1)
```

Comment:
This change make next two changes unnecessary (line 258 and 269) because, if fan control is set to be available (correctly), everything is ok and no additional checks are needed in actual get and set methods.